pkg/config/setup/config.go

```diff
@@ -974,6 +974,7 @@ func InitConfig(config pkgconfigmodel.Config) {
 	config.BindEnvAndSetDefault("installer.registry.url", "")
 	config.BindEnvAndSetDefault("installer.registry.auth", "")
 	config.BindEnv("fleet_policies_dir")
+	config.BindEnvAndSetDefault("__fleet_layers", []string{})
```

Comment:
Why is this config prefixed with `__` ?

Reply:
Because it's not a config we want customers to be able to set (it'll be overridden anyway). This is a config entry that is "reserved" to fleet policies so we can bring back up to the UI which policies are applied on the host.